README.md

```diff
@@ -199,6 +200,23 @@ application.
 
 A simple but secure example is detailed in `README.google_oauth2.md`.
 
+### DataDog Integration
+
+To get DataDog working you need to add the build pack [`heroku-buildpack-datadog`](https://github.com/deliveroo/heroku-buildpack-datadog) which enables the integration with roo_dd_bridge.
```

Comment:
That's not quite true, actually — `roo_dd_bridge` doesn't relate to the buildpack at all. It parses Heroku logs to push metrics to Datadog.